test/screenshotter/test.html

```diff
@@ -31,7 +31,35 @@
     <span id="math"></span>
     <span id="post"></span>
     <script type="text/javascript">
-      function handle_search_string(search, callback) {
+      function load_fonts(callback) {
+        if (!document.fonts || !document.fonts.load) {
+          callback();
+          return;
+        }
+
+        var results = [];
+        function load_font(name, style) {
+          // 10px is arbitrarily selected value
+          results.push(document.fonts.load((style || "") + " 10px " + name));
+        }
+
+        var fonts = {AMS:1,Caligraphic:3,Fraktur:3,Main:15,Math:12,Script:1,
+            SansSerif:7,Size1:1,Size2:1,Size3:1,Size4:1,Typewriter:1}; // see below
+        for (var f in fonts) {
```

Comment:
It would be nice if there was a way to know which fonts needed to be loaded for a particular test case so we don't have to always load all of them.  Not a big deal though.  I'd rather the tests be robust and consistent.

Reply:
@kevinbarabash Fortunately, we don't reload the page unless `reload` option is set.